linux_os/guide/system/accounts/accounts-restrictions/password_storage/accounts_password_last_change_is_in_past/rule.yml

```diff
@@ -0,0 +1,33 @@
+documentation_complete: true
+
+title: 'Ensure all users last password change date is in the past'
+
+description: |-
+    All users should have a password change date in the past.
+rationale: |-
```

Comment:
Please, include an empty line between the sections.

Reply:
:+1: changed in [135d663](https://github.com/ComplianceAsCode/content/pull/10031/commits/135d663c3dac8a18fb51088c0b758f095081c5fd)